unitTests/ui/ResponsiveComponents/ResponsiveComponentsManagerTest.ts

```diff
@@ -47,12 +47,21 @@ export function ResponsiveComponentsManagerTest() {
 
       it('should calls handle resize event when resize listener is called', () => {
         root.width = () => 400;
-        responsiveComponentsManager.register(responsiveComponent, root, 'id', component, {});
+        responsiveComponentsManager.register(responsiveComponent, root, 'foo', component, {});
```

Comment:
Why did we need to change `id` to `foo`?

Reply:
That was a mistake. Will change